src/app/modules/item/pages/item-by-id/item-by-id.component.ts

```diff
@@ -312,9 +313,10 @@ export class ItemByIdComponent implements OnDestroy, BeforeUnloadComponent, Pend
       filter(([ previous, current ]) => (!!current && (!previous || isTask(previous.item) || isTask(current.item)))),
       map(([ , current ]) => ensureDefined(current).item),
     ).subscribe(item => {
+      const isInitFromMobile = !this.layoutService.fullFrameValue.animated && this.breakpointObserver.isMatched('(max-width: 479px)');
```

Comment:
probably that logic should be moved the the layout service. This value of `479` is very layout specific and shouldn't be repeated at many places in the code. (you cannot have a single constant that you use both the css and ts?)

Reply:
The value `479` - fixed for ts. I've took standard (Handset: (max-width: 599.98px)) from https://material.angular.io/cdk/layout/overview#predefined-breakpoints and also changed it for css. But I can't fix quickly variable for css as `var()` won't work here.

Reply:
Layout better to do separated as after move it need to test a lot of cases